components/transactions/TransactionItem.js

```diff
@@ -161,7 +162,16 @@ const TransactionItem = ({ displayActions, collective, ...transaction }) => {
                 {displayedAmount.currency}
               </Span>
             </Container>
-            {hasOrder && <OrderStatusTag status={'PAID'} isRefund={isRefunded} fontSize="9px" px="6px" py="2px" />}{' '}
+            {hasOrder && (
+              <OrderStatusTag
+                status={'PAID'}
```

Comment:
It's not from this PR, but I'm surprised by this. Shouldn't that be:

```suggestion
                status={order.status}
```